FW/FreeRTOS/Task.cpp

```diff
@@ -0,0 +1,58 @@
+#include "FreeRTOS.h"
+#include "task.h"
+
+
+extern "C" {
```

Comment:
I don't think you usually indent code within an extern "C" block either. Come to think of it, maybe this file would make sense to make a .c file?